lib/internal/Magento/Framework/App/Cache/FlushCacheByTags.php

```diff
@@ -37,12 +37,6 @@ class FlushCacheByTags
      */
     private $tagResolver;
 
-    /**
```

Comment:
Please don't remove code blocks. If you want add the long class declarations back instead.

Reply:
Re-reviewing this, this was an erroneous removal as the type for `$cacheList` disappears.  I'll return this block.